src/test/java/org/jitsi/meet/test/PeerConnectionStatusTest.java

```diff
@@ -239,21 +209,21 @@ public void test2ndPeerInterruptedDuringCall()
 
         // Check also 2nd participant's local UI for indication
         MeetUIUtils.verifyLocalConnStatusIndication(
-                secondPeer, false /* disconnected */);
+            secondPeer, false /* disconnected */);
 
         // 4. Select remote again
         MeetUIUtils.selectRemoteVideo(owner, secondPeer);
 
         // Mute video(this is not real live scenario, but we want to make
         // sure that the grey avatar is displayed if the connection gets
         // disrupted while video muted)
-        new StopVideoTest("stopVideoOnParticipantAndCheck")
-            .stopVideoOnParticipantAndCheck();
+        //        new StopVideoTest("stopVideoOnParticipantAndCheck")
```

Comment:
why are those commented out (without any comment )? Is it for the work in progress ?

Reply:
I will check it, maybe I missed it while working on it.